### DISCUSSION TITLE
四月更新 - 3

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#1005] [通知]
## ✨ 重点

直播功能得到增强，支持显示分区，同时替换为移动端API

*如果你之前直播播放有问题，比如第二个视频黑屏之类的，请尝试这个版本，并在Github给个反馈，以便我确认该问题是否被修复*

## 🚀 功能

- 替换直播API，支持HEVC/仅听音频等功能 @Richasy (#1003)
    - 如果该直播支持HEVC，同时你设置了H265/HEVC优先，那么直播将选取HEVC源进行播放（请确保你的设备上安装了HEVC解码器）
    - 仅听音频，也就是黑听功能，会关闭视频流，可以节省流量，但可能对部分直播无效，这是正常现象。开启该功能后，音频封面将会盖住视频内容
- 支持显示直播分区 @Richasy (#994)
- 移除SYEngine @Richasy (#982)
    - 移除该组件可能会对视频卡顿有所帮助，有遇到这种问题的小伙伴可以帮忙测试一下

## 🐛 Bug 修复

- 修复用户视图滚动时条目丢失的问题 @Richasy (#1004)
- 避免搜索建议在搜索后才弹出 @Richasy (#999)
- 修复弹幕输入框焦点丢失的问题 @Richasy (#997)
- 修复互动视频返回异常 @Richasy (#996)
- 修复切换分P后字幕没有清理的问题 @Richasy (#995)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2204.3.0'>四月更新 - 3</a>.</em>

**Comment by HiRA017** [comment]
播放时依然掉帧、卡顿，全屏除了弹幕不显示任何内容；播放第二个视频直接黑屏。(显卡：RTX 3070 Laptop GPU/系统版本：Windows 11 22000.556）

**Reply by hez2010** [reply]
禁用掉了核显的话似乎就会出现这个问题，使用系统的UWP播放器放东西也不例外。

**Reply by HiRA017** [reply]
这样子噢… 那我关了独显直连再试试

**Reply by HiRA017** [reply]
关了确实有画面了，但还是掉帧卡顿，玄学

**Reply by Richasy** [reply]
掉帧的话可以尝试关闭G-Sync，或者开启/关闭垂直同步。这个东西确实不好整，解决掉帧问题有的说开启垂直同步，有的说关闭垂直同步，但总的来说，可能需要从显卡设置中找原因

**Reply by HiRA017** [reply]
最新测试结果：（我的笔记本没有G-Sycn）[关闭独显直连、NVIDIA控制面板开启Bili的垂直同步、选择RTX 3070]/[使用核显]：可以正常播放第一个视频（正常播放、正常最大化、无掉帧现象），但第二个视频依旧黑屏；[开启独显直连、NVIDIA控制面板开启Bili的垂直同步]：似乎可以正常播放第一个视频（但最大化无画面、有卡顿掉帧），第二个视频黑屏。

**Reply by Richasy** [reply]
我目前没什么好办法，手头上的四台设备都没出现过这种情况😢

**Reply by HiRA017** [reply]
确实很怪，只能暂时用用网页，不过会持续关注的。谢谢开发大佬的辛劳！

**Reply by Yan-nian** [reply]
3060一样的 没有开独显直连 第二个视频还是黑屏 就很怪
大佬加油
